### PR TITLE
Release version 1.3.0

### PR DESCRIPTION
**Pull request by raviks789**

### CODE REVIEW

File: module.info [hunk 1]
```diff
@@ -1,6 +1,6 @@
 Name: JIRA
-Version: 1.2.2
-Depends: monitoring (>=2.9.0), icingadb, ipl (>=0.11.0)
+Version: 1.3.0
+Depends: monitoring (>=2.9.0), icingadb (>= 1.0.0), ipl (>=0.11.0)
```
Comment (nilmerg) [inline module.info:3]: please use the new `Requires` syntax like [here](https://github.com/Icinga/icingaweb2-module-cube/blob/master/module.info)

**Review by lippserd — CHANGES_REQUESTED** [review]
Please use dark mode for screenshots and decide for proper dimensions, i.e. 1024x768. If you're using Chrome, use [this](https://chrome.google.com/webstore/detail/web-developer/bfbameneiokkgbdmiekhjnmfkcnldhhm). Then use the Chrome console commands to capture the screenshot.

File: doc/03-Configuration.md [hunk 2]
```diff
@@ -100,6 +104,28 @@ shown anywhere, it shouldn't disturb.
 
 ### Fill JIRA Custom Fields
 
+For your customized workflows you might need this module to ship additional
+fields. Additional jira custom fields for the issues can either be configured in Web GUI or manually:
```
Comment (nilmerg) [inline doc/03-Configuration.md:108]: `Jira`, uppercase first letter

File: doc/03-Configuration.md [hunk 3]
```diff
@@ -100,6 +104,28 @@ shown anywhere, it shouldn't disturb.
...
+
+1. **Web GUI** :
+   * Once Jira module has been enabled and its settings are configured. Go to `Templates Configuratioin`
```
Comment (nilmerg) [inline doc/03-Configuration.md:111]: *the* Jira module

Comment (nilmerg) [reply doc/03-Configuration.md:111]: ~~configured. Go to~~
configured, visit the

Comment (nilmerg) [reply doc/03-Configuration.md:111]: *Configuratio~~i~~n*

File: doc/03-Configuration.md [hunk 4]
```diff
@@ -100,6 +104,28 @@ shown anywhere, it shouldn't disturb.
...
+fields. Additional jira custom fields for the issues can either be configured in Web GUI or manually:
+
+1. **Web GUI** :
+   * Once Jira module has been enabled and its settings are configured. Go to `Templates Configuratioin` 
+page to create a new template. 
+![Web GUI - Template Configuration](screenshot/add_template.png)
+   * Click on the template to which the additional fields need to be added.
+![Web GUI - Edit Template](screenshot/template_config.png)
+   * Add the required additional fields to the template as shown in the screenshot below. These
+   fields can be associated with host group, service group or the custom variable. They could also be some other
```
Comment (nilmerg) [inline doc/03-Configuration.md:117]: *a* host group, service group or ~~the~~ custom variable

File: doc/03-Configuration.md [hunk 5]
```diff
@@ -100,6 +104,28 @@ shown anywhere, it shouldn't disturb.
...
+   values relevant to the issue being created. Currently only fields of type string, array, number and date are
+     supported in Web GUI and for multi-select array type and other types are not supported.
+![Web GUI - Add Field](screenshot/add_field.png)
+
+The fields added to a template will be listed in the `Fields List` tab of that corresponding template.
+![Web GUI - Fields List](screenshot/fields_list.png)
+ 
+2. **Manually** : Create corresponding templates in your `templates.ini` in 
+`<ICINGAWEB_CONFIGDIR>/modules/jira/templates.ini`. This template could
+subsequently be used while creating the issue.
```
Comment (nilmerg) [inline doc/03-Configuration.md:127]: ~~could subsequently~~
can then

File: doc/20-Issue-History.md [hunk 6]
```diff
@@ -26,17 +26,17 @@ comments posted to this issue will be shown. Also, there are and some links
 pointing to JIRA (directly to the issue or to it's project) and to the related
 Icinga Host or Service.
 
-![JIRA issue details](screenshot/issue_details.png)
+![JIRA issue details](screenshot/issue_detail.png)
 
 
-Monitoring module hook
+Monitoring / Icingadb module hook
```
Comment (nilmerg) [inline doc/20-Issue-History.md:32]:
```suggestion
Monitoring module / Icinga DB Web integration
```

**Review by nilmerg — CHANGES_REQUESTED** [review]

File: doc/03-Configuration.md [hunk 7]
```diff
@@ -16,9 +16,16 @@ Use `jira` as name.
 
 ## Configuration
 
-Currently you have to manually create a related configuration file. In future
-we'd love to allow you to provide these settings directly in the Web GUI. For now
-please create a dedicated module configuration directory, like:
+### Configuring in Web GUI
+
+Jira allows you to configure the settings in Web GUI. Once `jira` module has been enabled, go to `Configuration`
```
Comment (nilmerg) [inline doc/03-Configuration.md:21]: *the* .. module

File: doc/03-Configuration.md [hunk 8]
```diff
@@ -100,13 +107,30 @@ shown anywhere, it shouldn't disturb.
...
+fields. Additional Jira custom fields for the issues can either be configured in Web GUI or manually:
+
+1. **Web GUI** :
+   * Once the Jira module has been enabled and its settings are configured, visit the `Templates Configuration` 
+page to create a new template. 
+![Web GUI - Template Configuration](screenshot/add_template.png)
+   * Click on the template to which the additional fields need to be added.
+![Web GUI - Edit Template](screenshot/template_config.png)
+   * Add the required additional fields to the template as shown in the screenshot below. These
+   fields can be associated with host group, service group or custom variable. They could also be some other
```
Comment (nilmerg) [inline doc/03-Configuration.md:120]: **a** host group, ...

File: doc/03-Configuration.md [hunk 9]
```diff
@@ -100,13 +107,30 @@ shown anywhere, it shouldn't disturb.
...
+   values relevant to the issue being created. Currently only fields of type string, array, number and date are 
+   supported in Web GUI. Multi-select array type and other type custom fields are not supported.</br>
```
Comment (nilmerg) [inline doc/03-Configuration.md:122]: Please avoid `<br>`

**Review by nilmerg — CHANGES_REQUESTED** [review]
* Not all screenshots are 1024px wide, e.g. the hook previews
* Some screenshots are only correctly aligned on Github, may look different on the website
  Screenshot from 2023-03-14 17-34-29

**Review by lippserd — APPROVED** [review]

**Review by nilmerg — APPROVED** [review]

**Pull request merged**